packages/office-ui-fabric-react/src/components/Fabric/__snapshots__/Fabric.test.tsx.snap

```diff
@@ -505,6 +505,7 @@ exports[`Fabric renders a Fabric component in RTL 1`] = `
               -moz-osx-font-smoothing: grayscale;
               -webkit-font-smoothing: antialiased;
               color: #323130;
+              direction: rtl;
```

Comment:
I feel like I'm missing something but it seems `rtl` defaults to `undefined` in this PR and am not sure why it's showing up in this snapshot.

Reply:
this snapshot is for an RTL direction Fabric control

Reply:
That's what I was missing. :P